lib/config/validation.spec.ts

```diff
@@ -967,38 +967,19 @@ describe('config/validation', () => {
       expect(warnings).toHaveLength(1);
     });
 
-    it('validates valid customEnvVariables objects', async () => {
-      const config = {
-        customEnvVariables: {
-          example1: 'abc',
-          example2: 'https://www.example2.com/example',
-        },
-      };
-      const { warnings, errors } = await configValidation.validateConfig(
-        true,
-        config,
-      );
-      expect(warnings).toHaveLength(0);
-      expect(errors).toHaveLength(0);
-    });
-
-    it('errors on invalid customEnvVariables objects', async () => {
+    // adding this test explicitly because we used to validate the customEnvVariables inside repo config previously
+    it('warns if customEnvVariables are found in repo config', async () => {
       const config = {
         customEnvVariables: {
           example1: 'abc',
           example2: 123,
         },
       };
-      const { warnings, errors } = await configValidation.validateConfig(
-        true,
-        config,
-      );
-      expect(warnings).toHaveLength(0);
-      expect(errors).toMatchObject([
+      const { warnings } = await configValidation.validateConfig(false, config);
+      expect(warnings).toMatchObject([
         {
-          message:
-            'Invalid `customEnvVariables.customEnvVariables.example2` configuration: value is not a string',
           topic: 'Configuration Error',
+          message: `The "customEnvVariables" option is a global option reserved only for bot's global configuration and cannot be configured within repository config file`,
```

Comment:
```suggestion
          message: `The "customEnvVariables" option is a global option reserved only for Renovate's global configuration and cannot be configured within repository config file,`,
```

I think we should say `Renovate's` here, instead of `bot's`.

Also a sentence should end with a `.`. :wink: